### DISCUSSION TITLE
features.md 优化建议

### DISCUSSION THREAD

**Discussion by Geeyun-JY3** [the1812/Bilibili-Evolved#1405] [功能建议 / Ideas]
感觉可以往 [功能列表](features.md) 里加个类似于 [Previously part of Refined GitHub](https://github.com/sindresorhus/refined-github#previously-part-of-refined-github) 的章节，专门记录以前有现在 B 站已实现或有可用替代方案的功能

**Comment by LonelySteve** [comment]
想法不错，其实有 Git 提交历史在，随时可以做

**Comment by Geeyun-JY3** [comment]
或者干脆写进 README
